xml/System.Windows/DependencyObject.xml

```diff
@@ -1,782 +1,791 @@
+  
+-   Get, set, and clear utility methods for values of any dependency properties that exist on the <xref:System.Windows.DependencyObject>.  
+  
+-   Metadata, coerce value support, property changed notification, and override callbacks for  dependency properties or attached properties. Also, the <xref:System.Windows.DependencyObject> class facilitates the per-owner property metadata for a dependency property.  
+  
+-   A common base class for classes derived from <xref:System.Windows.ContentElement>, <xref:System.Windows.Freezable>, or <xref:System.Windows.Media.Visual>. (<xref:System.Windows.UIElement>, another base element class, has a class hierarchy that includes <xref:System.Windows.Media.Visual>.)  
+  
+   
+  
+## Examples  
+ The following example derives from <xref:System.Windows.DependencyObject> to create a new abstract class. The class then registers an attached property and includes support members for that attached property.  
+  
+ :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_Wpf/WPFAquariumSln/CSharp/WPFAquariumObjects/Class1.cs" id="Snippetdomain":::
+ :::code language="vb" source="~/samples/snippets/visualbasic/VS_Snippets_Wpf/WPFAquariumSln/visualbasic/wpfaquariumobjects/class1.vb" id="Snippetdomain":::  
+  
+ ]]></format>
+    </remarks>
+    <altmember cref="T:System.Windows.DependencyProperty" />
+  </Docs>
+  <Members>
+    <Member MemberName=".ctor">
+      <MemberSignature Language="C#" Value="public DependencyObject ();" />
+      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor() cil managed" />
+      <MemberSignature Language="DocId" Value="M:System.Windows.DependencyObject.#ctor" />
+      <MemberSignature Language="VB.NET" Value="Public Sub New ()" />
+      <MemberSignature Language="C++ CLI" Value="public:&#xA; DependencyObject();" />
+      <MemberType>Constructor</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>WindowsBase</AssemblyName>
+        <AssemblyVersion>3.0.0.0</AssemblyVersion>
+        <AssemblyVersion>4.0.0.0</AssemblyVersion>
+        <AssemblyVersion>5.0.0.0</AssemblyVersion>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <Parameters />
+      <Docs>
+        <summary>Initializes a new instance of the <see cref="T:System.Windows.DependencyObject" /> class.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+Automatic determination of current Dispatcher. Use alternative constructor
+                that accepts a Dispatcher for best performance.

```

Comment:
Is this remark relevant for the final user, or is it directed to code maintainers?